.github/ISSUE_TEMPLATE/z_release_major.md

```diff
@@ -54,7 +58,9 @@ Read the release notes for full details: {{url}}
 
 Following our support policy, versions {{versions}} are no longer supported. Projects running these versions should start planning to upgrade in the near future. For more details read our software support documentation: https://docs.communityhealthtoolkit.org/core/overview/supported-software/
 
-To see what's scheduled for the next releases have a read of the product roadmap: https://github.com/orgs/medic/projects?query=is%3Aopen+sort%3Aname-asc
+See what's scheduled for the next releases: https://github.com/orgs/medic/projects?query=is%3Aopen+sort%3Aname-asc
 ```
-- [ ] Announce the release on the [CHT forum](https://forum.communityhealthtoolkit.org/), under the "Product - Releases" category. You can use the previous message and omit `@channel`.
+- [ ] Announce the release on the [CHT forum](https://forum.communityhealthtoolkit.org/c/product/releases/26), under the "Product - Releases" category. You can use the previous message and omit `@channel`.
+- [ ] Announce the release of new documenation on the [CHT Forum](https://forum.communityhealthtoolkit.org/c/product/documentation/28), under the "Product - Documentation" category.
+- [ ] Schedule a Release communication call to educate stakeholders on product and documentation improvements
```

Comment:
Who should do this step?

Reply:
All of the steps in the "Communicating" section are for product management. Should this be clarified?

Reply:
Yeah that would be good. Maybe each heading should indicate who is to lead it?

Reply:
add "Product Manager" to the "Communicating" section. I'm less familiar with the other sections so will leave to you to update as makes sense.